futures-core/src/lib.rs

```diff
@@ -30,57 +30,8 @@ macro_rules! if_std {
     )*)
 }
 
-#[macro_export]
-macro_rules! pinned_deref {
-    ($e:expr) => (
-        $crate::core_reexport::mem::PinMut::new_unchecked(
-            &mut **$crate::core_reexport::mem::PinMut::get_mut_unchecked($e.reborrow())
-        )
-    )
-}
-
-#[macro_export]
-macro_rules! pinned_field {
```

Comment:
I removed this macro because it's an anti pattern. Better alternatives are:
- Usage of `PinMut::map_unchecked`
- `unsafe_pinned!` macro